### PR TITLE
Add ReactNativeImagePicker bindings

### PR DESCRIPTION
**Pull request by forki**
Bindings for https://github.com/marcshilling/react-native-image-picker
Needs https://github.com/fsprojects/Fable/pull/291

### CODE REVIEW

File: import/react-native-image-picker/Fable.Import.ReactNativeImagePicker.fs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+open Fable.Core
+open Fable.Import
+open Fable.Import.JS
+open Fable.Import.Browser
+
+module ReactImagePicker =
+
+    [<StringEnum>]
+    type CameraType =
+    | [<CompiledName("front")>] Front
```
Comment (alfonsogarciacaro) [inline import/react-native-image-picker/Fable.Import.ReactNativeImagePicker.fs]: By default, the first letter is lowered for `StringEnum` cases so you only need to use `CompiledName` if you actually **don't want** the first letter lowered (or you want to use a completely different name than the case name).

File: import/react-native-image-picker/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#r "node_modules/fable-core/Fable.Core.dll"
+#load "node_modules/fable-import-react/Fable.Import.React.fs"
+#load "node_modules/fable-import-react/Fable.Helpers.React.fs"
+#load "node_modules/fable-import-react/Fable.Import.ReactNative.fs"
+#load "node_modules/fable-import-react/Fable.Import.ReactNativeImagePicker.fs"
+
+open Fable.Core
+open Fable.Import
+module R = Fable.Helpers.React
+module RN = Fable.Helpers.ReactNative
```
Comment (alfonsogarciacaro) [inline import/react-native-image-picker/README.md]: This probably should be `module RN = Fable.Import.ReactNative`

**Comment by forki** [comment]
fixed the things from feedback and added sample code to readme

**Comment by forki** [comment]
fixed

**Comment by alfonsogarciacaro** [comment]
Awesome! 🔥

**Pull request merged**